Automattic/pocket-casts-android
# Sync background refresh setting

## Description

Add syncing of the global setting to refresh podcasts in the background.

## Testing Instructions

> [!note]
> This should be tested with staging. Use `debug` variant instead of `debugProd`.

1. Have two devices D1 and D2.
2. Make sure that both devices are synced before starting the test by refreshing apps in the profile.
3. D1: Go to `Profile`/`Settings (cog icon)`/`Storage & data use`.
4. D1: Toggle `Background refresh` setting `OFF`.
5. D1: Sync the device in the Profile.
6. D2: Sync the device in the Profile.
7. D2: Go to `Profile`/`Settings (cog icon)`/`Storage & data use`.
8. D2: `Background refresh` setting should be `OFF`.
9. D2: Toggle `Background refresh` setting `ON`.
10. D2: Sync the device in the Profile.
11. D1: Sync the device in the Profile.
12. D1: Go to `Profile`/`Settings (cog icon)`/`Storage & data use`.
13. D1: `Background refresh` setting should be `ON`.

You can test this with Wear as well but it might be hard setting up the staging environment. The change in `wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/settings/SettingsViewModel.kt` should be enough to confirm that it works.

## Checklist
 
#### I have tested any UI changes...
